### ISSUE TITLE
ELF: not found 

### ISSUE THREAD

**Issue by nvjacobo** [mattermost/mattermost-docker#387]
I have error "ELF: not found " on Raspberry Pi  3B  +

docker logs -f mattermostdocker_app_1
``
/mattermost/bin/mattermost: line 4: syntax error: unexpected “(”
Using existing config file /mattermost/config/config.json
Configure database connection…OK
Starting mattermost
/mattermost/bin/mattermost: line 1: ELF: not found
/mattermost/bin/mattermost: line 2: @%: not found
/mattermost/bin/mattermost: line 1: ��: not found
/mattermost/bin/mattermost: line 3: ����o���@Z
��@�
�@�@��@��@�@h�
�����@8: not found
/mattermost/bin/mattermost: line 4: syntax error: unexpected “(”

Client:
Version: 18.09.0
API version: 1.39
Go version: go1.10.4
Git commit: 4d60db4
Built: Wed Nov 7 00:57:21 2018
OS/Arch: linux/arm
Experimental: false

Server: Docker Engine - Community
Engine:
Version: 18.09.0
API version: 1.39 (minimum version 1.12)
Go version: go1.10.4
Git commit: 4d60db4
Built: Wed Nov 7 00:17:57 2018
OS/Arch: linux/arm
Experimental: false`

**Comment by pichouk** [comment]
Someone is having the same issue [on the Mattermost Peer-to-Peer forum](https://forum.mattermost.org/t/502-bad-gateway/7125/10), and I think it's because you are using an ARM device. The Mattermost Docker image is using the AMD64 binary, so I think it can't work on a Raspberry PI, sorry.

**Comment by nvjacobo** [comment]
> Someone is having the same issue [on the Mattermost Peer-to-Peer forum](https://forum.mattermost.org/t/502-bad-gateway/7125/10), and I think it's because you are using an ARM device. The Mattermost Docker image is using the AMD64 binary, so I think it can't work on a Raspberry PI, sorry.

Thanks @pichouk. Do you have plans for ARM support in the future?

**Comment by pichouk** [comment]
I think there is no ARM compatible binary built by Mattermost, so I think we will not have an ARM compatible Docker image, sorry.
I think you can find some information about this on [Mattermost Forum](https://forum.mattermost.org/t/compiling-on-arm/716).

**Comment by eratio08** [comment]
yes @pichouk is right the AMD64 binary release by mattermost which is used in the `app/Dockerfile` will not work with an ARM kernel.
You might take a look at https://gitlab.com/nerzhul/dockerfiles/blob/master/mattermost-server-arm64/Dockerfile how to build the binaries with go yourself. Sadly there is not docker image tag for arm provided by mattermost.

**Comment by nvjacobo** [comment]
thanks @eratio08
